webapp/channels/src/components/admin_console/cluster_table_container.tsx

```diff
@@ -43,7 +50,7 @@ export default class ClusterTableContainer extends React.PureComponent {
         }
     }
 
-    reload = (e) => {
+    reload = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
```

Comment:
```suggestion
    reload = (e: MouseEvent<HTMLButtonElement>) => {
```
We can just do this
MouseEvent is from react